packages/server/src/lib/superMode.ts

```diff
@@ -14,7 +15,11 @@ export interface SuperModeArticle {
 }
 
 export const fetchSuperModeArticles = async (): Promise<SuperModeArticle[]> => {
-    const records = await queryActiveArticles(stage, docClient);
+    const records = await queryActiveArticles(stage, docClient).catch(error => {
+        logError(`Error fetching super mode articles from dynamo: ${error}`);
+        putMetric('super-mode-error');
+        return [];
```

Comment:
this catch ensures that errors when fetching super mode data do not stop us from serving epics. But an alarm will be triggered